flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/AlterSchemaConverter.java

```diff
@@ -339,6 +364,10 @@ private void populatePrimaryKeyFromSourceTable(Schema oldSchema) {
             }
         }
 
+        private void populateDistribution(ResolvedCatalogTable oldTable) {
```

Comment:
follow naming pattern and call it `populateDistributionFromSourceTable`

Reply:
Changed.